src/main/app/claims/claimStoreClient.ts

```diff
@@ -69,7 +70,11 @@ export default class ClaimStoreClient {
       .get(`${claimStoreApiUrl}/${externalId}`)
       .then(claim => {
         if (claim) {
-          return new Claim().deserialize(claim)
+          if (userId !== claim.submitterId) {
```

Comment:
That breaks response journey! Defendant is unable to retrieve claim by external ID

Reply:
oops, :(

Reply:
how did the checks pass if it breaks them?

Reply:
I was thinking how the checks passed. How???

Reply:
@rlewan is investigating